src/rules/converters/use-default-type-parameter.ts

```diff
@@ -0,0 +1,13 @@
+import { RuleConverter } from "../converter";
+
+// See here for reference
+// https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unnecessary-type-arguments.md
```

Comment:
Would you mind removing these comments? Folks can generally find all the linked info on https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md. The other converters don't have them, and I don't want to start a precedent of adding extra stuff necessary for them - especially since the rule names sometimes change.

Reply:
Ack.